includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -686,95 +703,198 @@ public function stripe_modal_checkout_status( $status = 'enabled' ) {
+											$stripe_account_id
+										) :
+										esc_html__( 'Are you sure you want to disconnect GiveWP from Stripe?', 'give' );
+									$disconnect_url     = ( 'connect' === $details['type'] ) ?
+										give_stripe_disconnect_url( $stripe_account_id, $slug ) :
+										add_query_arg(
+											[
+												'post_type' => 'give_forms',
+												'page'    => 'give-settings',
+												'tab'     => 'gateways',
+												'section' => 'stripe-settings',
+												'give_action' => 'disconnect_manual_stripe_account',
+												'account' => $slug,
+											],
+											admin_url( 'edit.php' )
+										);
+
+									?>
+									<div id="give-stripe-<?php echo $slug; ?>" class="give-stripe-account-manager-list-item">
+										<div class="give-stripe-account-name-wrap">
+											<span class="give-stripe-account-name">
+												<?php _e( $account_name, 'give' ); ?>
+											</span>
+											<span class="give-field-description give-stripe-account-email">
+												<?php _e( $account_email, 'give' ); ?>
+											</span>
+											<span class="give-stripe-account-edit">
+												<?php if ( 'connect' !== $details['type'] ) { ?>
+												<a class="give-stripe-account-edit-name" href="#"><?php esc_html_e( 'Edit', 'give' ); ?></a>
+												<a
+													class="give-stripe-account-update-name give-hidden"
+													href="#"
+													data-account="<?php echo $slug; ?>"
+												><?php esc_html_e( 'Update', 'give' ); ?></a>
+												<a class="give-stripe-account-cancel-name give-hidden" href="#"><?php echo __( 'Cancel', 'give' ); ?></a>
+												<?php } ?>
+											<?php if ( $slug === $default_account ) { ?>
+												<span class="give-stripe-account-default give-stripe-account-badge">
+													<?php esc_html_e( 'Default Account', 'give' ); ?>
+												</span>
+											<?php } else { ?>
+												<span class="give-stripe-account-default">
+													<a
+														data-account="<?php echo $slug; ?>"
+														data-url="<?php echo give_stripe_get_admin_settings_page_url(); ?>"
+														class="give-stripe-account-set-default" href="#"
+													><?php esc_html_e( 'Set as Default', 'give' ); ?></a>
+												</span>
+											<?php } ?>
+
+											</span>
+										</div>
+										<div class="give-stripe-account-actions">
+											<span class="give-stripe-account-type-description give-field-description"><?php _e( 'Connection Method:', 'give' ); ?></span>
+											<span class="give-stripe-account-type-method"><?php echo give_stripe_connection_type_names( $details['type'] ); ?></span>
+											<?php
+											if (
+												$slug !== $default_account ||
+												(
+													is_array( $stripe_accounts ) &&
```

Comment:
@mehul0810 I already added a message related to this that `give_stripe_get_all_accounts` should return `array` and in this way, we can clear these extra checking.